test-data/unit/check-selftype.test

```diff
@@ -259,11 +259,11 @@ from typing import TypeVar, Type
 
 T = TypeVar('T', bound=str)
 class A:
-    def foo(self: T) -> T:   # E: The erased type of self 'builtins.str' is not a supertype of its class '__main__.A'
+    def foo(self: T) -> T:   # E: The erased type of self "builtins.str" is not a supertype of its class "__main__.A"
```

Comment:
Looking at these errors, I think we should switch them to using "short form", as we do for many other errors (in a _separate_ PR).

Reply:
Will do.